tsconfig.json

```diff
@@ -11,9 +11,9 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es2016" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+    "target": "es2022" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
IIRC, some client was using older version in their application, and hit some issues when newer version was here. I agree on using something more recent, but we may need to double-check if that won't break anything on the client side.